MaterialLib/Fluid/Viscosity/VogelsLiquidDynamicViscosity.h

```diff
@@ -0,0 +1,86 @@
+/*!
+   \file  VogelsLiquidDynamicViscosity.h
+   \brief Declaration of class for the pressure dependent viscosity model.
+
+   \copyright
+    Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+               Distributed under a Modified BSD License.
+               See accompanying file LICENSE.txt or
+               http://www.opengeosys.org/project/license
+ */
+#ifndef VOGELS_LIQUID_DYNAMIC_VISCOSITY_H_
+#define VOGELS_LIQUID_DYNAMIC_VISCOSITY_H_
+
+#include <string>
+#include <vector>
+#include <cmath>
+
+#include "MaterialLib/Fluid/FluidProperty.h"
+
+namespace MaterialLib
+{
+namespace Fluid
+{
+/// A temperature dependent viscosity model.
+/// <a href="ddbst">
+/// http://ddbonline.ddbst.de/VogelCalculation/VogelCalculationCGI.exe</a>
+class VogelsLiquidDynamicViscosity final : public FluidProperty
+{
+public:
+    /*!
+         \brief Viscosity defined by \f$e^{A+\dfrac{B}{C+T}}\f$
+         \param mat_id  0: Water
+                        1: Carbon dioxide
+                        2: Methane
+     */
+    explicit VogelsLiquidDynamicViscosity(const unsigned mat_id)
```

Comment:
Worst case: A struct for each material. Then the parameters also are not accessed via indices, which can make formulas easier to read.